### DISCUSSION TITLE
The Community Website

### DISCUSSION THREAD

**Discussion by chryzcode** [chryz-hub/chryz-hub.github.io#58] [General]
## Why?
- There is this belief. every real organization must have a good website
-  We have to present our self well
- We can use that to pull more people 

and so many more.

<b>So I and the team would have a short meeting, though we already had one earlier. Then the result of the meeting would be shared publicly and ideas would be appreciated generally.
### You can contribute to this by
 - Raising an issue, you see deem fit
 - Sharing ideas
 - Making a pull request

**Comment by mjvbz** [comment]
How can i become a member of this community ?

**Reply by chryzcode** [reply]
You can join the community by filling this [form](https://github.com/chryz-hub/chryz-hub.github.io/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) up
